app/apis/v1/record_time_api.rb

```diff
@@ -0,0 +1,22 @@
+module V1
```

Comment:
If you want to update or delete a specific record, you should identify what record you are going to delete or update.

I recommend you separate the API for creating and destroying the results.

Reply:
Seperated!